e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -65,4 +85,11 @@ private void assertThatAccessStructuredWasFetched(String conversationId, Documen
         assertThat(accessStructured.get("accessedAt")).isNotNull();
         assertThat(accessStructured.get("taskId")).isNotNull();
     }
+
+    private void assertThatAccessDocumentWasFetched(String documentId, Instant accessedAt, List<Document> accessDocuments) {
+        Document document = accessDocuments.get(0);
+        assertThat(document.get("objectName")).isEqualTo(documentId + ".json");
+        assertThat(document.get("accessedAt")).isNotEqualTo(accessedAt);
+        assertThat(document.get("taskId")).isEqualTo(TASK_ID);
```

Comment:
I would expect taskId to be the random uuid assigned by the adaptor and not this hardcoded value.

Reply:
Ok